interface/forms/functional_cognitive_status/new.php

```diff
@@ -142,97 +136,103 @@ function checkVal(id)
             });
         </script>
     </head>
-    <body class="body_top">
-        <div class="container">
+    <body>
+        <div class="container mt-3">
             <div class="row">
                 <div class="col-12">
                     <h2><?php echo xlt('Functional and Cognitive Status Form'); ?></h2>
                 </div>
```

Comment:
You do not need a row or a col-12 when it's just an h2 title. col-12 is only meant to allow the content to have a width of 100% and the h2 already does that by default in HTML5.

Reply:
OK, Thanks for your mentions 👍